docs/tutorials/installation.md

```diff
@@ -21,31 +20,34 @@ $ pip install "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/j
+| Google Cloud TPU                   | `pip install -U "jax[tpu]" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html`                                                                                       |
+| Apple ARM-based GPU (macOS)        | Follow [Apple's instructions](https://developer.apple.com/metal/jax/)                                                                                                                 |
+| AMD GPU                            | Use [Docker](https://hub.docker.com/r/rocm/jax) or [build from source](https://jax.readthedocs.io/en/latest/developer.html#additional-notes-for-building-a-rocm-jaxlib-for-amd-gpus)  |
+
 Continue reading this guide to learn about more options.
 
+(supported-platforms)=
 ## Supported platforms
 
 In the table below, check if your setup is supported. And if it says _"yes"_ or _"experimental"_ you can click on the corresponding link to learn how to install JAX in greater detail.
 
-|                  | Linux, x86_64            | Linux, aarch64     | macOS, Intel x86_64        | macOS, ARM-based           | Windows, x86_64 | Windows WSL2, x86_64        |
-|------------------|--------------------------|--------------------|----------------------------|----------------------------|-----------------|-----------------------------|
-| CPU              | [yes](#cpu)              | [yes](#cpu)        | [yes](#cpu)                | [yes](#cpu)                | [yes](#cpu)     | [yes](#cpu)                 |
-| NVIDIA GPU       | [yes](#nvidia-gpu)       | [yes](#nvidia-gpu) | no                         | n/a                        | no              | [experimental](#nvidia-gpu) |
-| Google Cloud TPU | [yes](#google-tpu)       | n/a                | n/a                        | n/a                        | n/a             | n/a                         |
-| AMD GPU          | [experimental](#amd-gpu) | no                 | no                         | n/a                        | no              | no                          |
-| Apple GPU        | n/a                      | no                 | [experimental](#apple-gpu) | [experimental](#apple-gpu) | n/a             |  n/a                        |
-
-Below are just some examples on how to install JAX (do consult with the detailed instructions provided further in this guide):
+|                  | Linux, x86_64            | Linux, aarch64        | macOS, Intel x86_64             | macOS, ARM-based                | Windows, x86_64  | Windows WSL2, x86_64            |
+|------------------|--------------------------|-----------------------|---------------------------------|---------------------------------|------------------|---------------------------------|
+| CPU              | [yes]{ref}`cpu`          | [yes]{ref}`cpu`       | [yes]{ref}`cpu`                 | [yes]{ref}`cpu`                 | [yes]{ref}`cpu`  | [yes]{ref}`cpu`                 |
```

Comment:
Note that the scope of these labels is global to the entire project, so it may be helpful to make them more informative (e.g. `installation-cpu` instead of `cpu`)